src/common/interpolation.c

```diff
@@ -907,6 +1064,21 @@ void dt_interpolation_compute_pixel4c(const struct dt_interpolation *itor, const
     *(__m128 *)out = _mm_set_ps1(0.0f);
   }
 }
+#endif
+
+void dt_interpolation_compute_pixel4c(const struct dt_interpolation *itor, const float *in, float *out,
+                                      const float x, const float y, const int width, const int height,
+                                      const int linestride)
+{
+  if(darktable.codepath.OPENMP_SIMD)
```

Comment:
not sure i understand the logic. the above function is not simd at all, right? it's called plain and that's exactly what it does, no vectorisation at all? so why would i want that called if OPENMP_SIMD is set?